simpleui-server/src/sui_data.ts

```diff
@@ -27,7 +28,7 @@ export class SuiData {
     static mockRequestNum = 0;
     static mockDataFileIndex = [];
     static uiProps = "";
-    static zmqMap = null;
+    static zmqMap: null|zmq_wrapper  = null;
```

Comment:
What exactly is this construction?

Reply:
The `zmqMap` is a map of zmq ports and their respective zmq_wrapper instances. This map is used to retrieve the sockets using the ports as their ID.

Reply:
the zmq_wrapper class just allows us to have some nice quality of life stuff. Such as having only a single SIGTERM event listener to close all the zmq sockets